Sming/Libraries/MPU6050/README.rst

```diff
@@ -0,0 +1,14 @@
+API Documentation
+-----------------
+
+.. doxygenclass:: MPU6050
+   :members:
+
+# MPU6050 Six-Axis (Gyro + Accelerometer)
+Based on code from [jrowberg/i2cdevlib](https://github.com/jrowberg/i2cdevlib/tree/master/ESP32_ESP-IDF/components/MPU6050) @ 605a740. Most of the code is the same, except:
+
+-   Removed MPU6050::ReadRegister function due to incompatibility. It is also not used anywhere in the original code.
+-   MPU6050_6Axis_MotionApps20.h and MPU6050_9Axis_MotionApps41.h are not included due to deps to freeRTOS. helper_3dmath.h is also not included since it is only used in the above mentioned files.
+-   Removed map function in favor of the Sming built-in one.
+-   Adapted include path, coding style and applied clangformat
+-   Deleted Calibration and Memory Block related code for code quality reason
```

Comment:
Have a look at other libraries/Components for general layout. Heading must be double-underlined (level 1) and give a good name for the library. For example:

```
MPU6050 Gyro / Accelerometer
============================
```

Follow this with the description. Note that `.rst` files (restructured text) syntax differs from markdown so remember to fix links, e.g.

```
`jrowberg/i2cdevlib<https://github.com/jrowberg/i2cdevlib/tree/master/ESP32_ESP-IDF/components/MPU6050>`
```

And put the API section last.

Reply:
Yeah, takes a bit of getting used to but has advantages to markdown! See https://sming.readthedocs.io/en/latest/information/develop/documentation.html